lib/cosmos/system/target.rb

```diff
@@ -149,7 +150,7 @@ def process_file(filename)
           rescue Exception => err
             raise parser.error(err.message)
           end
-          @requires << parameters[0]
+          @requires << filename
```

Comment:
The list of requires is now the full path name which is a change but matches what we do for cmd_tlm. Also I couldn't find anyone else using @requires anywhere.

Reply:
You are missing expanding the path for all cases.   Particularly in the line 144 case (just parameters[0]).  Need a filename = expand_path to be consisten.

Reply:
I added a test for requiring a file with an absolute path and it passes. Not sure what case you're thinking of.

Reply:
Line 144 is Cosmos.require_file(parameters[0])  - not using "filename".   Yet line 152 always adds "filename" to the @requires list

Reply:
Set filename inside the Cosmos.disable_warnings section.